content/applications/productivity/calendar/google.rst

```diff
@@ -153,3 +153,60 @@ Now, Odoo Calendar is successfully synced with Google Calendar!
 
    Events can be created in Google Calendar without sending a notification by selecting
    :guilabel:`Don't Send` when prompted to send invitation emails.
+
+Google OAuth FAQ
+================
+
+At times there can be misconfigurations that take place, and troubleshooting is needed to resolve
+the issue. Below are the most common errors that may occur when configuring the Google Calendar for
+use with Odoo.
+
+Production VS Testing Publishing Status
+---------------------------------------
+
+Choosing :guilabel:`Production` as the :guilabel:`Publishing Status` (instead of
+:guilabel:`Testing`) displays the following warning message:
+
+.. image:: google/published-status.png
+   :align: center
+   :alt: OAuth is Limited to 100 Sensitive Scope Logins.
+
+To correct this warning, navigate to the `Google API Platform
+<https://console.cloud.google.com/apis/credentials/consent>`_. If the :guilabel:`Publishing status`
+is :guilabel:`In Production`, click :guilabel:`Back to Testing` to correct the issue.
+
+No Test Users Added
+-------------------
+
+If no test users are added to the OAuth consent screen, then a :guilabel:`Error 403: access_denied`
```

Comment:
```suggestion
If no test users are added to the OAuth consent screen, then an :guilabel:`Error 403: access_denied`
```

Reply:
since the 'OAuth consent screen' is given a GUI below -- shouldn't *this one* be given the same formatting?